src/components/Breadcrumb/Breadcrumb.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Breadcrumb as CarbonBreadcrumb, OverflowMenu } from 'carbon-components-react';
+import overflowIcon20 from '@carbon/icons-react/lib/overflow-menu--horizontal/20';
+
+import BreadcrumbOverflowItem from './BreadcrumbOverflowItem';
+
+// TODO: set style process multi media
+const StyledBreadcrumb = styled(CarbonBreadcrumb)`
+  overflow: hidden;
+  text-overflow: ellipsis;
+`;
+
+// TODO: slash should not be click
+const StyledOverflowMenu = styled(OverflowMenu)`
+  &::after {
+    content: '/';
+    color: #171717;
+    margin-left: 0.5rem;
+  }
+  margin-right: 0.5rem;
+`;
+
+const StyledOverflowIcon20 = styled(overflowIcon20)`
+  & > circle {
+    cy: 26;
+  }
+`;
+
+const MINIMUM_OVERFLOW_THRESHOLD = 4;
+
+const propTypes = {
+  /** Pass in the BreadcrumbItem's for your Breadcrumb */
+  children: PropTypes.node,
+  /** Specify an optional className to be applied to the container node */
+  className: PropTypes.string,
+  /** Optional prop to omit the trailing slash for the breadcrumb */
+  noTrailingSlash: PropTypes.bool,
+  /** Define to show how many items */
+  threshold: PropTypes.number,
```

Comment:
`threshold` is the only prop change from the stock Breadcrumb component. All the others can be removed from proptypes and defaultprops